src/History/Command/UserActionHistoryCommand.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\History\Command;
+
+use App\Entity\Administrator;
+use App\History\UserActionHistoryTypeEnum;
+use Ramsey\Uuid\UuidInterface;
+
+class UserActionHistoryCommand
```

Comment:
Can you implement `AsynchronousMessageInterface` to handle this command asynchronously